src/xunit.performance.api/ETWProfiler.cs

```diff
@@ -156,7 +361,13 @@ private static void SetPreciseMachineCounters(IEnumerable<ProviderInfo> provider
                 }
 
                 if (profileSourceIDs.Count > 0)
+                {
+                    /*
+                     * FIXME: This function changes the -pmcsources intervals machine wide.
+                     *  Maybe we should undo/revert these changes!
+                     */
```

Comment:
[https://github.com/Microsoft/xunit-performance/issues/192](https://github.com/Microsoft/xunit-performance/issues/192)

---
In reply to: [101318647](https://github.com/Microsoft/xunit-performance/pull/191#discussion_r101318647) [](ancestors = 101318647)